source/features/preview-hidden-comments.tsx

```diff
@@ -23,13 +23,13 @@ const init = (): void => {
 		], details)!;
 
 		const reason = /was marked as ([^.]+)/.exec(header.textContent!)?.[1] ?? '';
-		if (!allowedReasons.has(reason)) {
+		if (forbiddenReasons.has(reason)) {
 			continue;
 		}
 
 		header.append(
 			<span className="Details-content--open">{header.firstChild}</span>,
-			<span className="Details-content--closed">{`${upperCaseFirst(reason)} — ${commentText}`}</span>
+			<span className="Details-content--closed">{(reason.length > 0 ? upperCaseFirst(reason) + ' — ' : '') + commentText}</span>
```

Comment:
Should be give up on showing the reason altogether? Since only off-topic comments can display it anyway.